### PR TITLE
order views with channels

### PR DESCRIPTION
**Pull request by AlicjaSzu**
I want to merge this change because...

**PR intended to be tested with API branch:** feature/multichannel-multicurrency

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Translated strings are extracted.
1. [x] Number of API calls is optimized.
1. [x] The changes are tested.
1. [x] Data-test are added for new elements.
1. [x] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### Test environment config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-139 User needs to be able to assign sales channel to draft order](https://app.clickup.com/t/2549495/SALEOR-139)

**Comment by patrys** [comment]
Task linked: [SALEOR-644 User needs to be able to activate and deactivate channel as wished](https://app.clickup.com/t/2549495/SALEOR-644)

File: src/channels/views/ChannelDetails/ChannelDetails.tsx [hunk 1]
```diff
@@ -46,6 +50,13 @@ export const ChannelDetails: React.FC<ChannelDetailsProps> = ({ id }) => {
     onCompleted: onSubmit
   });
 
+  const [activateChannel, activateChannelOpts] = useChannelActivateMutation({});
+
+  const [
+    deactivateChannel,
+    deactivateChannelOpts
+  ] = useChannelDeactivateMutation({});
```
Comment (orzechdev) [inline src/channels/views/ChannelDetails/ChannelDetails.tsx:58]: What about channel activation errors? Don't we want to handle them somehow?

Comment (AlicjaSzu) [reply src/channels/views/ChannelDetails/ChannelDetails.tsx]: I added notifications

**Review by orzechdev — APPROVED** [review]

**Pull request merged**